src/components/item/ListItem.vue

```diff
@@ -138,19 +142,20 @@ export default {
     align-items: inherit;
     margin-bottom: 10px;
     width: 100%;
-    cursor: pointer;
     transition: transform 0.2s;
 
     &.bordered {
       padding: 5px 15px;
       border: 1px solid map-get($colors, 'black');
 
-      .item-title {
-        font-size: 14px;
-      }
+      .item {
+        &-title {
+          font-size: 14px;
+        }
 
-      .item-details {
-        font-size: 12px;
+        &-details {
```

Comment:
bad idea to concat names like that - it's gonna be really hard to find that class after
so, if we use BEM we can separate between entities (block and element for example), but it's not a good idea to separate in the middle